internal/zinc-core/src/main/scala/sbt/internal/inc/IncrementalCommon.scala

```diff
@@ -149,11 +148,11 @@ private[inc] abstract class IncrementalCommon(
         invalidatedSources: Set[VirtualFile],
         classFileManager: XClassFileManager,
         pruned: Analysis,
-        override val previousAnalysis: Analysis,
```

Comment:
Is this change relevant to the fix?

Reply:
Nah, my thinking was to clean up as this parameter that was not necessary and introduced the confusion by duplication of parameters `previous` in `CycleState` vs `previousAnalysis` here.